src/helpers/gamepad_layouts/standard.ts

```diff
@@ -1,8 +1,6 @@
-// Holds layouts for different XBox official an clone controllers
+// Holds layouts for controllers officially supported by chromium
```

Comment:
Since this would now be the `standard` layout, maybe it's better to rename the variables that reference the buttons so it's not Xbox too.

Currently this file reads:

```ts
  const A = buttons[0],
    B = buttons[1],
    X = buttons[2],
    Y = buttons[3],
    // LB = buttons[4],
    // RB = buttons[5],
    // LT = buttons[6], // has .value
    // RT = buttons[7], // has .value
    // view = buttons[8],
    // menu = buttons[9],
    // L3 = buttons[10], // press left stick
    // R3 = buttons[11], // press right stick
    up = buttons[12],
    down = buttons[13],
    left = buttons[14],
    right = buttons[15],
    // XBOX = buttons[16],
    leftAxisX = axes[0],
    leftAxisY = axes[1],
    rightAxisX = axes[2],
    rightAxisY = axes[3]
```

Maybe we should rename them and add comments like:

```ts
  const button0 = buttons[0], // Xbox: A, PS: Cross
    button1 = buttons[1], // Xbox: B, PS: Circle
    button2 = buttons[2], // etc ...
    ...
```

Reply:
ok I gave them some better names